setup.py

```diff
@@ -5,10 +5,10 @@
 
 setuptools.setup(
     name='ocs_sample_library_preview',
-    version='0.3.6_preview',
+    version='0.4.0_preview',
     author='OSIsoft',
     license='Apache 2.0',
-    author_email='dendres@osisoft.com',
+    author_email='samples@osisoft.com',
```

Comment:
should this coincide with changing the pypi user 'osi-derekendres' to 'osi-samples' for consistency? @derekendres @dcbroad3

Reply:
Reverted